packages/codegen-ui-react/lib/__tests__/__snapshots__/studio-ui-codegen-react-forms.test.ts.snap

```diff
@@ -1897,7 +1933,7 @@ export declare type InputGalleryCreateFormInputValues<useBase extends boolean =
     attend?: UseBaseOrValidationType<useBase, boolean>;
     maybeSlide?: UseBaseOrValidationType<useBase, boolean>;
     maybeCheck?: UseBaseOrValidationType<useBase, boolean>;
-    arrayTypeField?: UseBaseOrValidationType<useBase, string>;
+    arrayTypeField?: UseBaseOrValidationType<useBase, string[]>;
```

Comment:
i think we run validation on each element in the array as opposed to passing the whole list

Reply:
Ok, I reverted the change to array types for now. If I find it needs changes, I'll PR that separately.